src/controllers/item.js

```diff
@@ -82,8 +84,10 @@ module.exports = {
     try {
       req.body.owner = req.user._id;
       await ItemModel.create(req.body);
+      logger.log('info', 'Item added successfully!');
       res.status(201).json({ message: 'Item created successfully.' });
     } catch (err) {
+      logger.log('error', err.message);
```

Comment:
Were these 2 logger statements used for testing? Then we should remove them.

Reply:
Yes, I will remove them now :+1: